com.toggl.TogglDesktop.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id": "com.toggl.TogglDesktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
```

Comment:
Does it work with 5.12?

Reply:
yes

Reply:
Why not using it then?

Reply:
Probably when I started writing the manifest, it was not out yet, I'm gonna change it right away, sorry. :)